extra_foam/gui/ctrl_widgets/pump_probe_ctrl_widget.py

```diff
@@ -65,6 +65,8 @@ def __init__(self, *args, **kwargs):
         self._abs_difference_cb.setChecked(True)
 
         self._reset_btn = QPushButton("Reset")
+        self.save_btn = QPushButton("Save to file")
```

Comment:
would rather prefer a QAction in a toolbar on this window instead of a button to save to files.

Reply:
Hmm, @CammilleCC I leave this up to you. I think either UI is acceptable.